runtime/compiler/env/VMJ9.h

```diff
@@ -1573,8 +1573,10 @@ class TR_J9VM : public TR_J9VMBase
    virtual TR_OpaqueClassBlock * getNullRestrictedArrayClassFromComponentClass(TR_OpaqueClassBlock *componentClass);
    virtual TR_OpaqueClassBlock * getLeafComponentClassFromArrayClass(TR_OpaqueClassBlock * arrayClass);
    virtual int32_t               getNewArrayTypeFromClass(TR_OpaqueClassBlock *clazz);
-   virtual TR_OpaqueClassBlock * getClassFromSignature(const char * sig, int32_t length, TR_ResolvedMethod *method, bool isVettedForAOT=false);
-   virtual TR_OpaqueClassBlock * getClassFromSignature(const char * sig, int32_t length, TR_OpaqueMethodBlock *method, bool isVettedForAOT=false);
+   TR_OpaqueClassBlock *getClassFromSignature(const char * sig, int32_t sigLength, TR_ResolvedMethod *method, bool isVettedForAOT=false);
```

Comment:
Why did you drop the virtual keyword from these functions?

Reply:
`TR_J9VM` is derived from `TR_J9VMBase` which is derived from `TR_FrontEnd` which defines
```
virtual TR_OpaqueClassBlock * getClassFromSignature(const char * sig, int32_t length, TR_ResolvedMethod *method, bool callSiteVettedForAOT=false);
```
So, the getClassFromSignature() from TR_J9VM is virtual anyway, but the `virtual` keyword makes it clear.
It it wasn't virtual we might run into problems because omr uses the generic TR_FrontEnd.

Reply:
That's right. I had forgotten about `TR_FrontEnd` since the `TR_J9VM` and `TR_J9VMBase` virtual methods aren't marked `override` like the other frontends.